tanjun/clients.py

```diff
@@ -812,7 +812,7 @@ def message_accepts(self) -> MessageAcceptsEnum:
     @property
     def is_human_only(self) -> bool:
         """Whether this client is only executing for non-bot/webhook users messages."""
-        return _check_human in self._checks
+        return _check_human in self.checks
```

Comment:
it'd be much cheaper to just cast _check_human to the type stored in self._checks